great-idea-website/css/index.css

```diff
@@ -77,31 +74,35 @@ h1, h2, h3, h4, h5 {
 .header{
```

Comment:
you should create a .container class to cover the whole project. This should be after the * { } and before the .header, then you can inside the container class margin: 0 auto; this will center your website rather than having it show up on the left of the page. inside the container class where you would also set the width of your whole website